r/R/install-arrow.R

```diff
@@ -137,3 +136,91 @@ reload_arrow <- function() {
     message("Please restart R to use the 'arrow' package.")
   }
 }
+
+
+#' Create an install package with all thirdparty dependencies
+#'
+#' @param outfile File path for the new tar.gz package. Defaults to
+#' `arrow_V.V.V_with_deps.tar.gz` in the current directory (`V.V.V` is the version)
+#' @param package_source File path for the input tar.gz package. Defaults to
+#' downloading from CRAN.
```

Comment:
Technically it will download from wherever `options(repos)` says, which might not be CRAN (like, you could do this with our nightly package repository too).